app/views/devise/sessions/_new.html.haml

```diff
@@ -1,5 +1,4 @@
-- content_for :extras do
-  = button_tag 'X', id: 'cancel_modal', class: "cancel-button"
+
 - content_for :modal_content do
   = render partial: 'login_form'
 
```

Comment:
why the 'x' button has been removed?

Reply:
Because we don't need it on this modal, The point is that you need to log in to access the site so you shouldn't be able to cancel out of it